table/tables/tables_test.go

```diff
@@ -761,3 +764,194 @@ func TestConstraintCheckForOptimisticUntouched(t *testing.T) {
 	err = tk.ExecToErr("commit")
 	require.Error(t, err)
 }
+
+func TestTxnAssertion(t *testing.T) {
+	store, clean := testkit.CreateMockStore(t)
+	defer clean()
+	// TODO: Locks produced by this test may be left without cleaning up, causing the test runs longer than expected.
```

Comment:
@MyonKeminta 
Would this case slow down the default ci a lot?

Reply:
Oh sorry. This comment can be removed, I think, since the related issue in client-go is fixed in my PR ([here](https://github.com/tikv/client-go/pull/311/files#diff-47a169599d0edd907d33f40b6bc32efa1615bbb0784d0260ee45b64c29cfa3adR408))

Reply:
@ekexium should this be fixed directly on this PR, or opening an new PR to the dev branch?

Reply:
@MyonKeminta I'd prefer to file a new PR to address the comments.